multisig/proposals/ethereum/vip-035/index.ts

```diff
@@ -0,0 +1,86 @@
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { ethereum } = NETWORK_ADDRESSES;
+
+export const sfrxETH = "0xac3E018457B222d93114458476f3E3416Abbe38F";
+export const VTREASURY = "0xFD9B071168bC27DBE16406eC3Aba050Ce8Eb22FA";
+export const vsfrxETH = "0x625d47820d26976DDeABee38784d2843ca95D8Ae";
+export const INITIAL_SUPPLY = parseUnits("1.2", 18);
+export const REWARDS_DISTRIBUTOR_XVS = "0x134bfDEa7e68733921Bc6A87159FB0d68aBc6Cf8";
+export const XVS = "0xd3CC9d8f3689B83c91b7B59cAB4946B063EB894A";
+export const XVS_REWARD_TRANSFER = parseUnits("2400", 18);
+export const ACM = "0x230058da2D23eb8836EC5DB7037ef7250c56E25E";
+export const SUPPLY_CAP = parseUnits("10000", 18);
+export const BORROW_CAP = parseUnits("1000", 18);
+export const RECEIVER = "0x6e74053a3798e0fC9a9775F7995316b27f21c4D2";
+
+export const SFrxETHOracle = "0x987010fD82FDCe099174aC605B88E1cc35019ef4";
+export const PRICE_DIFF = parseUnits("1.14", 18);
+
+export const vip035 = () => {
+  return makeProposal([
+    // Configure Oracle
+    {
+      target: ethereum.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [sfrxETH, [SFrxETHOracle, ethers.constants.AddressZero, ethers.constants.AddressZero], [true, false, false]],
+      ],
+    },
+    {
+      target: SFrxETHOracle,
+      signature: "initialize(address,uint256)",
+      params: [ACM, PRICE_DIFF],
+    },
+
+    // Add Market
+    {
+      target: VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [sfrxETH, INITIAL_SUPPLY, ethereum.NORMAL_TIMELOCK],
+      value: "0",
+    },
+    {
+      target: sfrxETH,
+      signature: "approve(address,uint256)",
+      params: [ethereum.POOL_REGISTRY, 0],
+      value: "0",
+    },
+    {
+      target: sfrxETH,
+      signature: "approve(address,uint256)",
+      params: [ethereum.POOL_REGISTRY, INITIAL_SUPPLY],
+      value: "0",
+    },
+    {
+      target: vsfrxETH,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["7200"],
+    },
```

Comment:
The liquidation fee for this market should be 1%, because the liquidation fee on this pool is 2%. We should set it in the VIP, as we made [here](https://github.com/VenusProtocol/vips/pull/304/files#diff-68ad1b6cb21bb92a79301fc07e4abd920a8afc2d247b16e49f42d6d9e898e3aeR67-R71) for rsETH